cis194/week2/zehua/src/Exercises.hs

```diff
@@ -0,0 +1,23 @@
+module Exercises where
+
+import Log
+
+-- ex1
+createLogMessage :: MessageType -> String -> [String] -> LogMessage
+createLogMessage mt ts msg = LogMessage mt (read ts :: TimeStamp) (unwords msg)
+
+parseWords :: [String] -> LogMessage
+parseWords ("I":t:xs)   = createLogMessage Info t xs
```

Comment:
You can get rid of `createLogMessage` and directly use `LogMessage` instead.   
If inline `unwords` bothers you, maybe you can put it in the `where` clause.   
As for `readXXX`, you can either do conventional pattern matching or use https://gitlab.haskell.org/ghc/ghc/wikis/view-patterns as @shineli1984 suggested.

Reply:
made some changes but still hoping to keep `createLogMessage` as a util function.